ports/micro-gl/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "micro-gl",
+  "version-date": "2024-06-18",
+  "description": "Fast, Super Slim, Embeddable, Headers only C++11 vector graphics library, that can run on any 32/64 bits computer without FPU or GPU. No standard library required.",
+  "homepage": "https://micro-gl.github.io/docs/microgl",
+  "license": "MIT"
```

According to the [upstream custom license](https://github.com/micro-gl/micro-gl/blob/master/LICENSE.MD), [should be null](https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#license) to indicate that users must read the copyright file.
```suggestion
  "license": ""
```